src/main/java/io/github/jhipster/online/web/rest/GitResource.java

```diff
@@ -119,12 +126,22 @@ public RedirectView callback(@PathVariable String gitProvider, String code) {
                         .INTERNAL_SERVER_ERROR);
             }
 
-            HttpHeaders headers = new HttpHeaders();
-            headers.add("user-agent", "Mozilla/5.0 (X11; Linux x86_64; rv:72.0) Gecko/20100101 Firefox/72.0");
```

Comment:
The user-agent was added to fix a Gitlab issue related to Cloudflare user-agent policies, I don't see it in the updated code https://github.com/jhipster/jhipster-online/issues/93#issuecomment-633696429 https://github.com/jhipster/jhipster-online/pull/165

Reply:
@ruddell : Thanks. I've added it as well. Although I've tested it without it and it seems to work. :thinking: